Modules/Microsoft365DSC/DSCResources/MSFT_TeamsGroupPolicyAssignment/MSFT_TeamsGroupPolicyAssignment.psm1

```diff
@@ -365,7 +365,7 @@ function Export-TargetResource
         $totalCount = $instances.Length
         foreach ($item in $instances)
         {
-            $Group = Find-CsGroup -SearchQuery $item.GroupId
+            $Group = Find-CsGroup -SearchQuery $item.GroupId -ExactMatchOnly $true
```

Comment:
I am trying to understand what the outcome would be here. If we have two instances with the same name, we should print a warning letting the user know we won't be exporting them like we do for other resource (e.g., AADApplication) instead of trying to determine which ones to export or skip.

Reply:
I update the PR to support this case.

Now we are using the original groupId (unique) and and use the first display name (are they are all exactly equals).
I think we don't need any warning with this behaviour.

WDYT ?